snuba/snuba_migrations/generic_metrics/0033_counters_meta_tag_values_table.py

```diff
@@ -0,0 +1,66 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import AggregateFunction, Column, DateTime, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget
+from snuba.utils.schemas import Float
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    granularity = "2048"
```

Comment:
We could start using a granularity of 8192 (the default of clickhouse). Lower granularity on tables which have lots of data maybe could come to bite us at some point later. I know the original tables have this lower granularity, but we can probably start fixing this on the new tables.

Reply:
I made a note of this. These tables are experimental anyways, so I can drop this table and remake it with better settings after we prove out the concept.